tests/test_Voting.py

```diff
@@ -0,0 +1,161 @@
+
+    usr = guild.members[0]
+    await dpytest.message(f"{KoalaBot.COMMAND_PREFIX}vote setChair {usr.id}")
+    dpytest.verify_message(f"Set chair to {usr.name}")
+    await dpytest.message(f"{KoalaBot.COMMAND_PREFIX}vote setChair")
+    dpytest.verify_message("Results will be sent to the channel vote is closed in")
+
+    await dpytest.message(f"{KoalaBot.COMMAND_PREFIX}vote addOption test option+test1")
+    dpytest.verify_message(f"Option test option with description test1 added to vote")
+    await dpytest.message(f"{KoalaBot.COMMAND_PREFIX}vote addOption test option 2+test2")
+    dpytest.verify_message(f"Option test option 2 with description test2 added to vote")
+
+    # await dpytest.message(f"{KoalaBot.COMMAND_PREFIX}vote send")
+    # dpytest.verify_message(f"Sent vote to 1 users")
+    # errors trying to send embed because I think an old version of discord.py is wanting it as a dict? not sure
+
+
+def test_two_way():
+    def test_asserts(f, *args, **kwargs):
+        try:
+            f(*args, **kwargs)
+        except AssertionError:
+            return
+        raise AssertionError
+
+    # test internal asserts don't false positive
+    t = Voting.TwoWay({1: 2, 3: 4})
+    t2 = Voting.TwoWay({1: 2, 2: 1, 4: 3})
+    assert t == t2
+
+    # test an invalid dict cannot be made
+    test_asserts(Voting.TwoWay, {1: 2, 2: 3})
+
+    def ta2():
+        t = Voting.TwoWay()
+        t[1] = 2
+        t[2] = 3
+        test_asserts(ta2)
+
+
+def test_vote_manager_general():
+    vm = Voting.VoteManager(db_manager)
+    assert not vm.active_votes
+    vote = vm.create_vote(ctx, "Test Vote")
+    added = db_manager.db_execute_select("SELECT * FROM votes WHERE author_id=?", (ctx.author.id,))
+    assert added
+    assert ctx.author.id in vm.active_votes.keys()
+    assert vm.get_vote(ctx) == vote
+    assert vm.has_active_vote(1234)
+    vm.cancel_vote(1234)
+    cancelled = db_manager.db_execute_select("SELECT * FROM votes WHERE author_id=?", (ctx.author.id,))
+    assert not vm.has_active_vote(1234)
+    assert not cancelled
+
+
+def test_vote_general():
+    vm = Voting.VoteManager(db_manager)
+    vote = vm.create_vote(ctx, "Test Vote")
+    assert vote.id == ctx.author.id and vote.guild == ctx.guild.id and vote.title == "Test Vote"
+    assert not vote.is_ready()
+
+    vote.add_role(7890)
+    assert 7890 in vote.target_roles
+    roles = db_manager.db_execute_select("SELECT * FROM vote_target_roles WHERE vote_author_id=?", (ctx.author.id,))
+    assert roles[0][1] == 7890
+    vote.remove_role(7890)
+    assert 7890 not in vote.target_roles
+    roles = db_manager.db_execute_select("SELECT * FROM vote_target_roles WHERE vote_author_id=?", (ctx.author.id,))
+    assert not roles
+
+    vote.set_vc(1234)
+    assert vote.target_voice_channel == 1234
+    vc = db_manager.db_execute_select("SELECT * FROM votes WHERE author_id=?", (ctx.author.id,))
+    assert vc[4] == 1234
```

Comment:
Errors on my machine and Travis (another reason for having smaller tests is we now don't know if future tests pass)